README.md

```diff
@@ -641,6 +641,39 @@ These steps are performed once per deployment (multiple times per project).
 
 9. `_refresh`
 
+### 3.2.3 Transition Amazon SES resource out of sandbox
+
+Perform these steps once the cloud insfrastructure has been provisioned for the
+shared deployment, section #3.3. Before continuing, make sure that the SES
```

Comment:
If that step has to be done first, why does this section precede it in the README?

Reply:
I thought it made sense to include it as part of the `3.2 One-time manual configuration of deployments` section, since this needs to be done only once after creating the resource and it's a manual step.